lib/internal/bootstrap_node.js

```diff
@@ -302,13 +302,13 @@
 
   function setupProcessFatal() {
 
-    process._fatalException = function(er) {
+    process._fatalException = function(er, fromPromise) {
       var caught;
 
       if (process.domain && process.domain._errorHandler)
         caught = process.domain._errorHandler(er) || caught;
 
-      if (!caught)
+      if (!caught && !fromPromise)
```

Comment:
What’s the reasoning for this? That it’s already made visible by `process.on('unhandledRejection')`?